src/puppetlabs/puppetdb/cli/anonymize.clj

```diff
@@ -8,9 +8,9 @@
             [puppetlabs.puppetdb.catalogs :as catalogs]
             [puppetlabs.puppetdb.cheshire :as json]
             [puppetlabs.puppetdb.cli.export :refer [export-metadata-file-name]]
-            [puppetlabs.puppetdb.cli.import :refer [parse-metadata]]
+            [puppetlabs.puppetdb.cli.import :as import]
             [puppetlabs.puppetdb.schema :as pls]
-            [puppetlabs.puppetdb.utils :refer [export-root-dir add-tar-entry]]
+            [puppetlabs.puppetdb.utils :as utils :refer [export-root-dir add-tar-entry]]
```

Comment:
can we just use the utils alias for everything? looks like we're already there with add-tar-entry. I try to avoid using :as and :refer in the same line.